homeassistant/components/shelly/coordinator.py

```diff
@@ -475,6 +475,11 @@ def sw_version(self) -> str:
 
     async def _async_disconnected(self) -> None:
         """Handle device disconnected."""
+        # Sleeping devices send data and disconnects
```

Comment:
disconnect

Reply:
Going to replace https://github.com/thecode/core/blob/34ae51d9942439332ac6f97603327e6c7ee8892a/homeassistant/components/shelly/coordinator.py#L480 
with a property (did not want to change that in a cherry-pick), I'll fix it there